eclipse-collections/src/main/java/com/fasterxml/jackson/datatype/eclipsecollections/EclipseCollectionsDeserializers.java

```diff
@@ -183,7 +198,18 @@
             TypeDeserializer elementTypeDeserializer,
             JsonDeserializer<?> elementDeserializer
     ) throws JsonMappingException {
+        return findBeanDeserializer(type, config, beanDesc);
+    }
 
+    @Override
+    public JsonDeserializer<?> findMapLikeDeserializer(
+            MapLikeType type,
+            DeserializationConfig config,
+            BeanDescription beanDesc,
+            KeyDeserializer keyDeserializer,
+            TypeDeserializer elementTypeDeserializer,
+            JsonDeserializer<?> elementDeserializer
+    ) throws JsonMappingException {
```

Comment:
Delegation here loses some information about possible custom key and value deserializers, as well as type deserializers (needed for polymorphic handling of values). Method called can of course try to figure out these details on its own but that's quite a bit of work.

Reply:
Since none of the Map types are actually reported as map-like at this point (and I don't wanna patch it into 2.11) it's a bit of a moot point. Right now the value type deser is determined directly from the JavaType like you say in RefValueHandler.java and EclipseMapDeserializers.java:274.

How about removing the addition of this maplike method from this PR since it's superfluous and I make a second PR against 2.12 for map-like?

Reply:
I trust you to know this well enough so no objections.